src/logic/wca-api.js

```diff
@@ -22,7 +26,7 @@ const wcaApiFetch = (userToken, path, fetchOptions = {}) => {
     Object.assign({}, fetchOptions, {
       headers: new Headers({
         Authorization: `Bearer ${userToken}`,
-        'Content-Type': 'application/json',
+        "Content-Type": "application/json",
```

Comment:
The precommit hook made this change. This option isn't specified on the config file, so it's going with the default: https://github.com/prettier/prettier/blob/2.8.0/docs/options.md#quotes.

(I can bypass the precommit hook if that's preferred.)